docs/tutorials/google/start.ipynb

```diff
@@ -394,13 +392,10 @@
    "source": [
     "valid_qubit = device.qubits[0]\n",
     "\n",
-    "# Re-map qubits and gates for Sycamore hardware.\n",
-    "syc_circuit = cg.optimized_for_sycamore(\n",
-    "    circuit,\n",
-    "    new_device=device,\n",
-    "    qubit_map=lambda q: valid_qubit) # Constant fn since there's only one qubit.\n",
+    "# Transform circuit to use an available hardware qubit.\n",
+    "hw_circuit = circuit.transform_qubits(lambda q: valid_qubit)\n",
```

Comment:
While I agree with not using `optimize_for_gateset` since it breaks WYSIWYG, I worry that removing gateset considerations entirely from this tutorial will leave readers stranded as soon as they attempt something outside the HW gateset.

CC @tanujkhattar to recommend a WYSIWYG-friendly transformer for this, if one exists.

Reply:
Instead of using a transformer, we could point them to further documentation for non-native gates.  Either here or in the "Next steps" section at the end.

Reply:
We currently have a next step "Explore [best practices](https://github.com/quantumlib/Cirq/blob/master/docs/google/best_practices.md) for getting circuits to run on hardware." That seems like it would be a natural places to talk about transformers that can compiler to native gates. It currently talks about validation and `optimized_for_sycamore`, so this would need to be updated to reflect recent development.

Reply:
+1 to Matthew's recommendation - if we update that doc to cover this detail (here or in a follow-up), I'd be happy to sign off on the changes here.

Reply:
I updated the test to talk about `transform_qubits` instead of `optimized_for_sycamore` and added a note about transforming to available gates with a link to the best practices guide. I couldn't see any guides about transformers or compilation target gatesets, but once we have something we can link to that instead.